### ISSUE TITLE
Empty page if no bridges available (zola)

### ISSUE THREAD

**Issue by westerguard** [matrix-org/matrix.org#1735]
On the [bridges page](https://matrix.turbo.fish/ecosystem/bridges/), you've got all the bridges, but heir are also some empty one. Like Mastodon and Tox. These link to an [empty page ](https://matrix.turbo.fish/ecosystem/bridges/mastodon/). 

I understand the behavior (no bridges=empty page) but I feel like the page should at least mention that there aren't any bridges available.
Or there should be no link on the bridge overview if there are no bridges. 
Or we just need a Mastodon and Tox bridge 😄

**Comment by thibaultamartin** [comment]
Good catch again! For Mastodon I'm going to change it to ActivityPub and add Kazarma as soon as I get an ACK from their author, and for Tox it's going to be fixed once I import the data of the obsolete projects. I'll leave the issue open while it's still a problem.

Thanks!
